src/global.h

```diff
@@ -22,7 +22,7 @@
   2013-Dec-27 Removed QUICK_FS option.					TJB
   2014-May-20 Added ref_veg_vegcover.					TJB
 **********************************************************************/
-char *version = "4.2.1 2014-December-21";
+char *version = "4.2.2 2015-January-23";
```

Comment:
See comment about version numbering. I think the bug fix numbering scheme is inconsistent with 4.X (even if that is what we want for 5.X)